odoo/addons/base/tests/test_format_address_mixin.py

```diff
@@ -0,0 +1,58 @@
+from odoo.addons.base.tests.test_views import ViewCase
+
+
+class FormatAddressCase(ViewCase):
+    def assert_address_view(self, model):
+        # pe_partner_address_form
+        address_arch = """<form><div class="o_address_format"><field name="city"/></div></form>"""
+        address_view = self.View.create({
+            'name': 'view',
+            'model': model,
+            'arch': address_arch,
+            'priority': 900,
+        })
+
+        # view can be created without address_view
+        form_arch = """<form><field name="id"/><div class="o_address_format"><field name="street"/></div></form>"""
+        view = self.View.create({
```

Comment:
Shouldn't we give a priority of 1, to be sure ``get_view`` herebelow correctly returns the one we expect ?

Reply:
Arf, indeed.